straxen/plugins/veto_events.py

```diff
@@ -0,0 +1,126 @@
+import strax
+import straxen
+
+import numpy as np
+import numba
+from immutabledict import immutabledict
+
+
+@strax.takes_config(
+    strax.Option('event_left_extension_nv', default=0,
+                 help="Extends events this many ns to the left"),
+    strax.Option('event_resolving_time_nv', default=300,
+                 help="Resolving time for fixed window concidene [ns]."),
+    strax.Option('event_min_hits_nv', default=6,
+                 help="Minimum number of fully convinet hitlets to define an event."),
+    strax.Option('gain_model_nv',
+                 help='PMT gain model. Specify as (model_type, model_config)'),
+    strax.Option('n_nveto_pmts', type=int,
+                 help='Number of TPC PMTs'),
```

Comment:
Can you add the default values? Also, why do you want to have n_nveto_pmts and channel_map both, seems a bit redundant?

Reply:
I one could argue the same for the TPC. I think I started using it in the other plugins to be consistent with the TPC. But I agree, since I have to call the channel map anyhow I can remove those options. (Does not matter anymore, but this option was also supposed to be not tracked.)

Reply:
The TPC has a 1T version support, the NV does not ;)

Reply:
I am currently waiting for their implementations into CMT. I would not merge before that.

Reply:
thanks, that sounds good